odk1-src/briefcase-using.rst

```diff
@@ -204,9 +217,25 @@ To get help about the command line operation:
 Pulling form data from Aggregate
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. code-block:: console
+- CLI flag: `-plla` or `--pull_aggregate`
+- Usage:
+
+  .. code-block:: console
+
+      $ java -jar {path/to/briefcase-jar-file} --pull_aggregate --form_id {form-id} --storage_directory {path/to/briefcase-storage-location} --aggregate_url {aggregate-url} --odk_username {username} --odk_password {password}
 
-  $ java -jar {path/to/briefcase-jar-file} --pull_aggregate --form_id {form-id} --storage_directory {path/to/briefcase-storage-location} --aggregate_url {aggregate-url} --odk_username {username} --odk_password {password}
+- Help section:
+
+  .. code-block:: console
```

Comment:
If we don't find more appropriate/semantic way to document these,
they can stay in code blocks like this.

But the highlighter on the `code-block` should (I *think*) be `none`, not `console`.

Reply:
I see more `code-block`s with `console` in lots of other pages. Do you mean that these particular blocks should be `none` or that `console` might be an invalid option for a `code-block`?

Reply:
`console` is the syntax highlighter for Bash terminal sessions

Reply:
I understand. I'll change it! :)